docs/tutorials/get-started.md

```diff
@@ -29,4 +29,32 @@ At this point, all the relevant environments required by Pipcook have been insta
 
 ```sh
 $ pipcook run examples/pipelines/mnist-image-classification.json
-```
\ No newline at end of file
+```
+
+### install docker image
+
+First instal [Dockerfile](https://github.com/alibaba/pipcook/blob/master/Dockerfile) to local.
+
+open Dockerfile's path and install image.
+
+```sh
+$ git clone https://github.com/alibaba/pipcook.git && cd pipcook
+$ docker build -t taobao/pipcook .
+```
+
+Check the image after installed.
+
+```sh
+$ docker images
+```
+
+| REPOSITORY | TAG | IMAGE ID | CREATED | SIZE |
+| --- | --- | --- | --- | --- |
+| taobao/pipcook | latest | c297c73d62d4 | 6 hours ago | 3.67GB |
```

Comment:
It's better to print the output from docker command, and wrap the above in code block.

Reply:
Done.